lab.js

```diff
@@ -0,0 +1,325 @@
+// // }
+
+// // console.log(average(array2))
+
+
+
+
+// console.log("Problem 6")
+// let string1 = "Today is a good day";
+// let object = {};
+
+
+// const frequencyMap = string =>{
+//     for (let i = 0; i < string.length;i++){
+//         if (object[string[i]]){
+//             object[string[i]] += 1;
+//         }
+//         else {
+//             object[string[i]] =1
+//         }
+//     }
+//     return object
+// }
+
+// console.log(frequencyMap(string1))
+
+
+
+
+
+// // function frequencyMap(arr1) {
+// //     for (let i = 0; i < string.length; i++) {
+// //         if (object[string[i]]) {
+// //             object[string[i]] += 1
+// //         } else {
+// //             object[string[i]] = 1
+// //         }
+// //     }
+// //     return object
+// // }
+// // console.log(frequencyMap(string))
+
+
+
+// console.log("Problem 7")
+// const fizzBuzz = (num1,num2) =>{
+//     for (i=num1; i<=num2;i++){
+//         if (i%3===0 && i%5===0){
+//             console.log("FizzBuzz!!!!!!!!!!!")
+//         }
+//         else if (i%3===0){
+//             console.log("Fizz")
+//         }
+//         else if (i%5===0){
+//             console.log("Buzz")
+//         }
+//         else {
+//             console.log(i)
+//         }
+//     }
+//     return fizzBuzz
+// }
+
+// console.log(fizzBuzz(1,16))
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+// // function fizzBuzz(num1, num2) {
+// //     for (let i = num1; i <= num2; i++) {
+// //         if (i % 3 === 0 && i % 5 === 0) {
+// //             console.log("FizzBuzz")
+// //         } else if (i % 3 === 0) {
+// //             console.log("Fizz")
+// //         } else if (i % 5 === 0) {
+// //             console.log("Buzz")
+// //         } 
+// //         else {
+// //             console.log(i)
+// //         }
+// //     }
+// // }
+// // fizzBuzz(1,27)
+
+// console.log("Problem 8")
+
+
+// let inputTwo = [1,2,3,4,5]
+// let inputOne = 1
+
+// const valueIsGreaterThanAverageOfArr = (input1, input2)=>{
+//     let sum =0;
+//     for (let i =0; i < input2.length;i++){
+//         sum += input2[i]
+//     }
+//     let avg = sum/input2.length;
+//     let double1 = input1 *2;
+// return double1>avg
+// }
+
+// console.log(valueIsGreaterThanAverageOfArr(inputOne,inputTwo))
+
+
+
+
+
+// let input2 = [1,2,3,4,5];
+// function valueIsGreaterThanAverageOfArr(input1, input2){
+//     let sum = 0;
+//     for (i = 0; i<input2.length; i++){
+//         sum += input2[i]
+//     };
+//     let avg = sum/input2.length;
+//     let double = input1 * 2; 
+//     console.log(double > avg ? "true" : "false");
+// }
+
+// valueIsGreaterThanAverageOfArr(1,input2)
+
+
+console.log("Problem 9")
+
+let arr = [36,35,49,30,68,38,52,48,20,10]
+
+const  secondSmallArr = input =>{
+    let smallest = arr[0];
+    let secondSmall = arr[0]; 
+    for (let i=0; i<input.length;i++){
+        if (smallest > arr[i]){
+            secondSmall = smallest;       
+            smallest = arr[i];
+        }
```

Comment:
you would need the else statement in case arr[i] is smaller than secondSmallest but larger than smallest